src/metabase/query_processor/expand.clj

```diff
@@ -307,7 +307,7 @@
       :next    (recur f  1 unit))
     (let [f (datetime-field f unit)]
       (cond
-        (core/= n  0) (= f (value f (relative-datetime :current)))
+        (core/= n  0) (= f (value f (relative-datetime  0 unit)))
```

Comment:
@moumny, what's the reason behind the changes in this file? Just so I can better understand the PR